.github/workflows/performance.yaml

```diff
@@ -1,11 +1,9 @@
 name: MIGraphX Performance Tests
 
 on:
-  push:
-    branches: [develop]
-
   pull_request:
     branches: [develop]
+    types: [opened, synchronize, closed]
```

Comment:
Do we want to run on closed  PR ?

Reply:
there's another script that checks if closed+merged vs closed